### PR TITLE
User Profile content updates

### PR DESCRIPTION
**Pull request by katypies**
## Ticket

Resolves
* #305 

## Changes

- Updated the content for Account Management to reflect changes to the User Profile
- Removed content about contact info on the Domain Management page since contact information is no longer managed there.

## Context for reviewers

* updated content from the [Google Doc](https://docs.google.com/document/d/1oxKgGR6mrVSQEwmrB9kNrhrhKAQ4-9fw0xxCQoT_qxc/edit#heading=h.gjdgxs)

### CODE REVIEW

File: pages/help/help_account-management.md [hunk 1]
```diff
@@ -60,19 +58,46 @@ If you don’t have another domain manager, email <help@get.gov> to let us know
...
- 
+Updating this information in your profile won’t affect your Login.gov account information.
+
+### Email address
+
+To update your email address, visit your account on Login.gov. 
+
+We recommend **using a Login.gov account that’s only connected to your work email address** and no other address. However, we recognize that may not be flexible enough for each use case. Here are three options that describe how to ensure your desired email address is the one associated with your registered domains:
+
+**Option 1: Sign in with a different email address that’s associated with your Login.gov account**
```
Comment (h-m-f-t) [inline pages/help/help_account-management.md:82]:
```suggestion
#### Option 1: Sign in with a different email address that’s associated with your Login.gov account
```

File: pages/help/help_account-management.md [hunk 2]
```diff
@@ -60,19 +58,46 @@ If you don’t have another domain manager, email <help@get.gov> to let us know
...
+
+A Login.gov account may have multiple email addresses associated with it, but your .gov profile only reflects the one you use to sign in. If the email address listed on your .gov profile is not the one you’d like to use: 
+1. Navigate to your [Login.gov account](https://secure.login.gov/account/connected_accounts){.usa-link--external} and verify that your desired email address is listed. If it’s not listed, add it.
+2. Then, sign in to the [.gov registrar](https://manage.get.gov/){target="_blank"} using your desired email address.
+3. After signing in to the registrar, you can immediately sign out. That’s all it takes to change the email address associated with your profile. However, on future sign-ins, you’ll need to use that same email address. 
+
+**Option 2: Remove other email addresses from your Login.gov account**
```
Comment (h-m-f-t) [inline pages/help/help_account-management.md:89]:
```suggestion
#### Option 2: Remove other email addresses from your Login.gov account
```

File: pages/help/help_account-management.md [hunk 3]
```diff
@@ -60,19 +58,46 @@ If you don’t have another domain manager, email <help@get.gov> to let us know
...
+
+You can remove other email addresses associated with your Login.gov account, leaving only your desired email address. However, that may impact other government services you access using your Login.gov account. (If that’s not acceptable, you should consider Option 3 below.) To remove other email addresses:
 
+1. Navigate to your [Login.gov account](https://secure.login.gov/account/connected_accounts){.usa-link--external} and review “Your connected accounts”.
+2. Verify that you’re comfortable with these accounts having access to a single email address.
+3. Add your desired email address if it’s not already listed.
+4. Delete any other email addresses. 
 
+**Option 3: Create a new Login.gov account using only your desired email address**
```
Comment (h-m-f-t) [inline pages/help/help_account-management.md:98]:
```suggestion
#### Option 3: Create a new Login.gov account using only your desired email address
```

File: pages/help/help_account-management.md [hunk 4]
```diff
@@ -60,19 +58,46 @@ If you don’t have another domain manager, email <help@get.gov> to let us know
...
 
+1. Create a new [Login.gov account](https://login.gov/create-an-account/){.usa-link--external} and add your desired email address. 
+    - If this email address is associated with another Login.gov account, you’ll need to remove it from that account first.
+2. Then, sign in to the [.gov registrar](https://manage.get.gov){target="_blank"} and verify your identity. Identity verification is required each time a new Login.gov account accesses the registrar.
+3. After you’ve signed in, email <help@get.gov> and we’ll associate your new account with your domain.
```
Comment (h-m-f-t) [inline pages/help/help_account-management.md:103]:
```suggestion
3. After you’ve signed in, [contact us](../../contact) and we’ll associate your new account with your domain.
```
I think we intentionally used the email address here, but I disagree with my former self.

**Review by h-m-f-t — CHANGES_REQUESTED** [review]
Excellent work! I've confirmed that `#update-your-contact-information-email-phone` isn't linked to in manage.get.gov or anywhere else you haven't already updated. 

A few small changes to make and then this is ready.

File: pages/help/help_account-management.md (hunk 4 above)
Comment (katypies) [reply pages/help/help_account-management.md:103]: @h-m-f-t There's one other instance where we ask people to email us (if they deleted their login.gov account) - want to fix that one and point to the contact form as well?

Comment (h-m-f-t) [reply pages/help/help_account-management.md:103]: Yeah, I agree.

**Review by h-m-f-t — APPROVED** [review]

**Pull request merged**